### PR TITLE
Copy link for sharing

### LINKED ISSUE #73: Add social share feature for restaurants
**What is your feature?**
Add a social share button to the results card. The button should open a module that has the following functions: copy link, share to email, share to Facebook, and share to Twitter. The share features should share the maps link and the website of a restaurant.

**Why is this feature needed?**
Users need a way to share their selected restaurant with friends.

**Who is your feature for?**
Users of MunchMatch. 

**Additional context**

- [x] Create social share component #74 
- [ ] Add sharing functionality
    - [x] Add copy link functionality
    - [ ] Add email functionality
   - [ ] Add Facebook and Twitter functionality

### PR DESCRIPTION
**Pull request by arielyxw**
Clicking the copy link button copies the restaurant's google link to the user's clipboard.

Partially resolves #73.

### CODE REVIEW

File: munch-match/src/components/ShareModal/ShareModalContent.jsx [hunk 1]
```diff
@@ -4,52 +4,65 @@ import ShareButton from "./ShareButton";
 
 const ShareModalContent = ({ imageUrl, name, address, shareUrl }) => {
     const handleCopyLink = () => {
+        const linkField = document.querySelector(`.${styles.linkField}`);
```
Comment (boxy8) [inline munch-match/src/components/ShareModal/ShareModalContent.jsx]: Don't need this can just write the `shareUrl` prop to clipboard

Comment (arielyxw) [reply munch-match/src/components/ShareModal/ShareModalContent.jsx:7]: It copies "undefined" without that assignment

Comment (arielyxw) [reply munch-match/src/components/ShareModal/ShareModalContent.jsx]: Yes ma'am have fixy now

**Review by boxy8 — APPROVED** [review]
Nice, LGTM 😄

**Pull request merged**